src/components/ThreadEnvelope.vue

```diff
@@ -21,7 +21,7 @@
 
 <template>
 	<div>
-		<div>
+		<div style="display: flex">
```

Comment:
No inline please, just give the div a proper class

Reply:
i didnt know how to call it. how about `leftandrigh` :smiley:

Reply:
How about something like thread--header :)